portfolio/src/main/webapp/reviews.html

```diff
@@ -36,38 +36,41 @@
     </div>
 
     <section class="section section-dark">
+      <div class="sign">
+        <span class="fast-flicker">Gi</span>mm<span class="flicker">e</span>!
+      </div>
+      <br><br><br>
       <div class="user-review-container">
         <form action="/reviews" method="POST">
           <div class="row">
-            <div class="col-25">
-              <label for="reviewer-name">Name</label>
-            </div>
-            <div class="col-75">
+            <div class="col">
               <input type="text" id="reviewer-name" name="reviewer-name" placeholder="Your name..">
             </div>
           </div>
           <div class="row">
-            <div class="col-25">
-              <label for="reviewer-input">Review</label>
-            </div>
-            <div class="col-75">
-              <textarea id="reviewer-input" name="reviewer-input" placeholder="Review here.." style="height:200px"></textarea>
+            <div class="col">
+              <textarea id="reviewer-input" name="reviewer-input" placeholder="  Review here.." style="height:200px"></textarea>
             </div>
           </div>
           <div class="row">
             <input type="submit" value="Submit" class="button submit-button">
           </div>
         </form>
       </div>
+    </section>
 
-      <label for="max-comments"># of Comments (between 1 and 5):</label>
+    <section class="section section-dark">
+      <div class="sign">
+        <span class="fast-flicker">S</span>e<span class="flicker">e</span>!
+      </div>
+      <br><br><br>
```

Comment:
Usually a code smell if you need more than one br or p. Could we use margin here instead?